package.json

```diff
@@ -11,15 +11,22 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+		"eject": "react-scripts eject",
+		"cy:open": "yarn start --silent & cypress open"
   },
   "eslintConfig": {
-    "extends": "react-app"
-  },
+		"extends": "react-app",
+		"globals": {
+			"cy": "cy"
+		}
+	},
```

Comment:
Code format is off.

Reply:
Not sure why it looks like that here, but in vsc it looks good.

Reply:
Yes, that IS weird. It’s fixable though...